### ISSUE TITLE
nvidia-driver-390 plymouth-start.service not starting after updates on 5/27

### ISSUE THREAD

**Issue by nathanblaubach** [pop-os/pop#300]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="18.04 LTS"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 18.04 LTS"
VERSION_ID="18.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
xserver-xorg-video-nvidia-390:
  Installed: 396.24-0ubuntu1~pop1
  Candidate: 396.24-0ubuntu1~pop1
  Version table:
 *** 396.24-0ubuntu1~pop1 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu bionic/main amd64 Packages
        100 /var/lib/dpkg/status
     390.48-0ubuntu3 500
        500 http://us.archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages

**Issue/Bug Description:**

On startup, my system goes into a loop of the following:

Jun 09 11:40:29 pop-os systemd[1]: Started Show Plymouth Boot Screen.
Jun 09 11:40:30 pop-os systemd[1]: plymouth-start.service: Start request repeated too quickly.
Jun 09 11:40:30 pop-os systemd[1]: plymouth-start.service: Failed with result 'start-limit-hit'.
Jun 09 11:40:30 pop-os systemd[1]: Failed to start Show Plymouth Boot Screen.

Eventually, it breaks and just starts. My NVIDIA X Server Settings no longer displays any informaiton about installed drivers.

I have re-installed the os and confirmed that the driver works properly before applying the updates.

This issue affects both my laptop:
Host: N53SV 1.0 
CPU: Intel i7-2630QM (8) @ 2.900GHz 
GPU: NVIDIA GeForce GT 540M 

and my desktop:
CPU: Intel i7-3770K (8) @ 6.300GHz
GPU: NVIDIA GeForce GTX 550 Ti

On the laptop, this is the only behavior that is noticeable. On my desktop, My desktop completely freezes after about 15 minutes of uptime.

**Steps to reproduce (if you know):**

This started happening when I updated my PC on 5/27. Looking at the updates around that time, I would have to guess that it had to do with this update:
xserver-xorg-video-nvidia-390
	Transitional package for xserver-xorg-video-396
	Version: 396.24-0ubuntu1-pop1

**Expected behavior:**

Driver behaves the same

**Other Notes:**

Let me know if there is anything else that I can do to help. Love the OS!

**Comment by blaubachn** [comment]
I also just did a fresh install of the non-nvidia iso, and this issue does not affect my system when I do that.

**Comment by mmstick** [comment]
Is `pop-desktop` installed?

**Comment by blaubachn** [comment]
Yep. That and everything else that comes as a part of the nvidia iso

**Comment by BobyMCbobs** [comment]
I have had a similar problem. My graphics drivers sometimes wouldn't work (sometimes it would on reboot). I had to reinstall to fix (I did try and fix prior to doing so), I installed Synaptic and forced the drivers to be not installed. Now I'm having up update packages through Synaptic to avoid breaking my rig again.

**Comment by blaubachn** [comment]
My issue is more specifically with just the above driver. I guess I should have been more clear about that. The Nvidia ISO downloads the 390 driver automatically for me, then when I do the updates I mentioned before, the driver stops functioning properly and disappears from the pop!_shop. I am able to run my system just fine with the 340 driver, but that defeats the purpose of using the Nvidia ISO.
